### DISCUSSION TITLE
@Mike777Khoo 还请麻烦帮我申请退款，谢谢

### DISCUSSION THREAD

**Discussion by TangAssos** [getlantern/lantern-forum-cn#634] [Q&A]
首先， 我的心情很压抑，在很久很久之前，安卓机刚开始流行，江湖就有传闻 蓝灯 是很OK的VPN，辗转多年，生活条件允许我购买付费版的VPN时，第一时间购买了蓝灯，谁成想，手机无法使用，电脑无法授权，安装了很多版本仍旧如此，各种不省心与不开心交汇着，在这，能肯定的是@Mike777Khoo管理员一直都在帮大家处理问题，可是，这是一个团队的事情，他负责的内容也许之是反馈和记录，谢谢给了我激活码，当我满怀欣喜激活成专业版时，马上就很失望，还请协助我申请退款，谢谢，附图有付款记录，再次感谢。

![蓝灯3](https://github.com/getlantern/lantern-forum-cn/assets/173048245/11cbc2c2-8b21-43dc-a4b8-0ea6958d53e5)
![蓝灯2](https://github.com/getlantern/lantern-forum-cn/assets/173048245/9187e419-e10c-4f46-af0b-9fa321bc16e9)
![蓝灯1](https://github.com/getlantern/lantern-forum-cn/assets/173048245/d24a4285-a776-4e69-b3dd-b3d5c8e6c1b3)
![蓝灯4](https://github.com/getlantern/lantern-forum-cn/assets/173048245/e62c49b6-4ef7-49e2-a152-5fb20b2ca763)

**Comment by MaxMKMax** [comment]
@TangAssos 您好. 谢谢您对我的认可. 很抱歉我们的app无法让您满意. 我会帮您提交退款申请. 谢谢您之前对蓝灯的支持. 希望您以后有机会再回来用蓝灯. 谢谢您!

**Reply by TangAssos** [reply]
@Mike777Khoo 蓝灯团队为有你这么负责的接线反馈员而感到荣幸，再次真诚的谢谢你协助我退款。

**Comment by MaxMKMax** [comment]
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
为了保护您的隐私. 请您在这里提交您的个人信息来申请退款. 您也可以在帖子删掉您的支付单据. 如果下周没有人帮您跟进, 您再回来找我帮忙. 谢谢您!

**Reply by TangAssos** [reply]
已提交成功，谢谢

**Reply by TangAssos** [reply]
@Mike777Khoo 管理员你好，退款申请提交数日了，一直未收到退款，请你协助我退款，谢谢

**Reply by MaxMKMax** [reply]
@TangAssos 
您好. 客服那里应该已经收到退款申请了. 退款一般需要 10- 15 天来完成. 谢谢您的耐心等待. 如果您过了 15 天没收到, 再来找我. 我帮您再追问看看. 谢谢!
